components/ContributorCard.js

```diff
@@ -86,7 +86,10 @@ const ContributorCard = ({ intl, width, height, contributor, currency, ...props
           </H5>
         </LinkContributor>
         <StyledTag my={2} padding="5px" letterSpacing="0.05em" fontStyle="initial">
-          {formatMemberRole(intl, getMainContributorRole(contributor))}
+          {formatMemberRole(intl, getMainContributorRole(contributor)) == 'Financial Contributor' &&
```

Comment:
This won't work because the strings are internationalized. For example with a French user the line `formatMemberRole(intl, getMainContributorRole(contributor))` will return `Contributeur flinancier`.

What you can rely on is `getMainContributorRole(contributor)` because it returns an actual role from the `roles` enum.

Reply:
Made changes